examples/shopify/tailwind.config.js

```diff
@@ -0,0 +1,21 @@
+const defaultTheme = require("tailwindcss/defaultTheme");
+
+/** @type {import('tailwindcss').Config} */
```

Comment:
Does this type specifier work even though there's an empty line between the object definition and the comment? Also, does this work when doing just an assignment? I would've though it only worked when declaring new variables

Reply:
<img width="882" alt="CleanShot 2022-08-16 at 14 27 28@2x" src="https://user-images.githubusercontent.com/20950876/184965402-040ac465-3a76-45ee-b62a-10cd751e2f79.png">
It does work, but I'll delete the line